### DISCUSSION TITLE
Where is all the backend code?

### DISCUSSION THREAD

**Discussion by Infiniti20** [plausible/analytics#1013] [Self-Hosted Support] [answered]
I just want to take a look at how it was built. I think it's a beautiful, and functional app, so I want to create something similar. I just want to see how you structure the backend.

**Comment by Vigasaurus** [comment, marked as answer]
It's primarily in the `lib` directory, with some specific backend-adjacent things in `priv`, the Reactive front-end is in `assets`, and the rest of the directory structure (config, tests, etc) should be pretty straightforward.
